backend/README.md

```diff
@@ -30,3 +31,19 @@ npm start
 ```
 
 The port on which the server is listening is reported in the console output.
+
+## Environment variables
+Here's the list of the envirnment variables that could to be configured.
+
+```
+process.env.DATABASE_URL
+```
+- stores the url to the database. If the variable is missing, the backend connects to our test database. 
+
+```
+process.env.FOOSBOT_TOKEN
+process.env.FOOS_CHANNEL_NAME
+```
+- these variables are used by the slackbot to announce the results of the matches You can obtain the bot Token on slack settings page and then select the channel you'd like to deploy the bot on. 
```

Comment:
You're missing a period after "matches".

Reply:
fixed